### DISCUSSION TITLE
Profiles?

### DISCUSSION THREAD

**Discussion by mntbighker** [ghostty-org/ghostty#4109] [Feature Requests, Ideas] [answered]
Are there any plans to add profiles (like bookmarks) with tagging? In my case in iTerm2 I use the context menu to open all remote sessions with a given tag, and then in most cases I sh+cmd-I to enter the same commands in all tabs in the selected window.

**Comment by mitchellh** [comment, marked as answer]
There are so many other requests for this. Please search.
